src/ferc_xbrl_extractor/helpers.py

```diff
@@ -2,6 +2,20 @@
 
 import logging
 
+import pandas as pd
+
+
+def parse_dates(date_str: str) -> pd.Timestamp:
+    """Helper to normalize date strings/parse in a consistent way."""
+    try:
+        if "24:00:00" in date_str:
+            return pd.to_datetime(
+                date_str.replace("24:00:00", "00:00:00")
+            ) + pd.Timedelta(days=1)
+        return pd.to_datetime(date_str)
+    except ValueError:
+        return pd.NaT
```

Comment:
This is nice and compact.

Do you understand how we were getting valid timestamps before without doing any of this munging?

Reply:
I think that the only column that actually was being properly turned into a `datetime` is the `publication_time` because that gets parsed separately from the RSS feed metadata, not XBRL directly. Other date columns were just getting written as strings to SQLite.